backend/app/assets/javascripts/spree/backend/admin.js

```diff
@@ -159,7 +159,6 @@ $(document).ready(function(){
       return ui;
   };
 
-  $('table.sortable').ready(function(){
     var td_count = $(this).find('tbody tr:first-child td').length
```

Comment:
no problem at all!, thanks both for reviewing!

The `have_title` suggestion was great, and I've gone about to actually change all those breadcrumb expectations to `have_title`, as when I was fixing those tests the other day, it took me a while to understand where "PromotionsPromotion" comes from.

All changed now.